<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>731</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber_6">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Hex</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLCDNumber" name="lcdNumber_3"/>
    </item>
    <item row="2" column="0">
     <widget class="QLCDNumber" name="lcdNumber_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber_5">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Hex</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>DECIMAL</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>HEXA</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumber_4">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Hex</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_8">
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Bin</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_9">
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Bin</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_7">
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Bin</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>BINARIO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ROJO</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>VERDE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>AZUL</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
